src/main/java/org/jabref/gui/citationkeypattern/GenerateCitationKeyAction.java

```diff
@@ -45,8 +47,14 @@ public void execute() {
 
         checkOverwriteKeysChosen();
 
-        BackgroundTask.wrap(this::generateKeys)
-                      .executeWith(Globals.TASK_EXECUTOR);
+        if (!this.isCanceled) {
+            BackgroundTask backgroundTask = this.generateKeysInBackground();
+            backgroundTask.showToUser(true);
+            backgroundTask.titleProperty().set(Localization.lang("Autogenerate citation keys"));
+            backgroundTask.messageProperty().set(entries.size() + " " + Localization.lang("entries"));
```

Comment:
```suggestion
            backgroundTask.messageProperty().set(Localization.lang("%0 entries", entires.size()));
```

See https://devdocs.jabref.org/getting-into-the-code/code-howtos#using-localization-correctly 😇

Reply:
Ah, I understand. Should have read your whole message before starting to 'fix' it. I now put "%0/%1 entries" so the progress is reflected there as well, but then this would be messed up in other languages I guess. I'll revert that part.

Reply:
Also, I think I did it wrong anyways, as '%0/%1 entries' is not a localization key.

Reply:
As @Siedlerchr pointed out, neither is '%0 entries'. Don't know why but I was sure it was there... I added '%0/%1 entries' in 
3b0dfed.